### DISCUSSION TITLE
GitHub is going to falsely suspend my account

### DISCUSSION THREAD

**Discussion by artonson** [community/community#30813] [New to GitHub] [answered]
I'm simply working from my office in Munich when an email came saying 
```
Hello,

You are receiving this email because your GitHub account appears to be associated with an entity sanctioned under US Trade Control laws. As a result, at 12:01 a.m. ET on September 1, 2022 your account will be suspended, and you will no longer be able to log into your account or access its contents. Please wind down use of your account before that time.

If you believe this is a mistake and you are not, or are no longer, associated with any sanctioned entity, you can appeal this determination by submitting a request here: https://airtable.com/shrfI0fsZxp9SleGQ.

Our team will review your appeal and determine if we are able to remove the restrictions.
```
I've filed an appeal as https://support.github.com/ticket/personal/0/1750654 but have no idea if it has got any attention. 

I would really appreciate if someone from GitHub would get this matter resolved. I will be soon unable to work using this account.

**Comment by Neozxc** [comment]
Did you try to appeal a determination by submitting a request on that URL that they've provided you? At the end of paragraph two.

**Reply by artonson** [reply]
Thank you! 
Forgot to mention: I've actually submitted an appeal request and also opened an issue under my personal account info.

**Comment by queenofcorgis** [comment, marked as answer]
Hi there!

We're not able to discuss account specifics on here, but thank you for submitting your Reinstatement Request. 

Please be patient and someone will get back to you as soon as they are able with any information etc. Submitting more requests will only slow any response.

As I say, please exercise patience and someone will be in touch on the email address you fill in on the reinstatement form.

**Reply by artonson** [reply]
Hi! Thank you! What happens if no one will be able to reach the issue by the time the account is scheduled to be suspended?

**Comment by ghost** [comment]
did you get your account back? i recently suspended too, i've submitting reinstatement request about 3 weeks ago and still no answer.
